pypi.md

```diff
@@ -0,0 +1,41 @@
+# Publishing Python packages to PyPI
+
+PyPI - the Python Package Index - is a repository of software for the Python programming language. The production instance of PyPI lives at https://pypi.python.org/pypi. A pre-production instance of a newer interface to PyPI can currently be found at https://pypi.org/.
+
+As a developer, when you type the command `pip install something`, by default that `something` comes from PyPI. When we create Python packages that might be generally useful to the public (for example, Wagtail plugins), it's nice to consider publishing them to PyPI so others can easily find and use them.
+
+## Authoring
+
+Before trying to create a new package for PyPI, it might be useful to review the [official Python documentation](https://packaging.python.org/tutorials/distributing-packages/) about packaging and distributing projects.
+
+Publishing a package generally consists of first creating a distribution file from your source code, using a `setup.py` file, and then 
+publishing that distribution file to PyPI, using [twine](https://pypi.python.org/pypi/twine).
+
+(In the past, these steps could be accomplished with a single command - `python setup.py upload` - [but this is no longer recommended for security reasons](https://github.com/pypa/twine#why-should-i-use-this).)
+
+A simple set of commands to accomplish this looks like:
+
+1. `python setup.py sdist bdist_wheel` (creates both a source distribution and a [Wheel archive](https://pip.pypa.io/en/stable/reference/pip_wheel/) of your project, and places them in a `dist` subdirectory.
+2. `twine upload dist/*.whl` (pushes your new distribution files to PyPI)
+
+The contents of your project's `setup.py` file determine how your code is packaged and how it gets published on PyPI. There are a number of useful and important fields that you should include in this file:
+
+- `url`: link to GitHub repository.
+- `author`: should be CFPB.
+- `author_email`: should be tech@cfpb.gov.
```

Comment:
Did we decide on `tech@cfpb.gov` over the `devops@cfpb.gov` address?

Reply:
I left this as `tech@` to keep consistency with currently published projects.

Reply:
I spoke with @chosak offline. We decided to move forward with `tech@cfpb.gov` as recommended here.